package.json

```diff
@@ -79,5 +80,8 @@
     "webpack": "^5.68.0",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.7.4"
+  },
+  "resolutions": {
```

Comment:
What it is done for? Is it instead of changes in yarn.lock file?

Reply:
To use correct loaders version. Without resolutions deck.gl dependency is 3.2 that doesn't have necessary change. We need beta version.